File: frontend/src/constants/article.ts
// 与 backend/internal/constants/article.go 保持一致
export const ArticleStatus = {
  DRAFT: 'draft',
  PUBLISHED: 'published',
  SCHEDULED: 'scheduled',
} as const;

export const ArticleStatusLabels: Record<string, string> = {
  [ArticleStatus.DRAFT]: '草稿',
  [ArticleStatus.PUBLISHED]: '已发布',
  [ArticleStatus.SCHEDULED]: '定时发布',
};

File: backend/internal/constants/article.go
package constants

// ArticleStatus 文章状态枚举（README 枚举出现位置清单必列）。
const (
	ArticleDraft     = "draft"     // 草稿
	ArticlePublished = "published" // 已发布
	ArticleScheduled = "scheduled" // 定时发布
)

// ArticleStatuses 全部文章状态。
var ArticleStatuses = []string{ArticleDraft, ArticlePublished, ArticleScheduled}
